migrator/migrations/m_213_to_m_214_populate_deployment_containers_imageidv2/migration_test.go

```diff
@@ -0,0 +1,167 @@
+//go:build sql_integration
+
+package m213tom214
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/migrator/migrations/m_213_to_m_214_populate_deployment_containers_imageidv2/schema"
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/suite"
+)
+
+type migrationTestSuite struct {
+	suite.Suite
+
+	db  *pghelper.TestPostgres
+	ctx context.Context
+}
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(migrationTestSuite))
+}
+
+func (s *migrationTestSuite) SetupSuite() {
+	s.ctx = sac.WithAllAccess(context.Background())
+	s.db = pghelper.ForT(s.T(), false)
+	pgutils.CreateTableFromModel(s.ctx, s.db.GetGormDB(), schema.CreateTableDeploymentsStmt)
+}
+
+func (s *migrationTestSuite) TestMigration() {
```

Comment:
Add something similar to the `existingDB` option in the process indicator migration's test. This will allow scale testing  the migration using a large database.

https://github.com/stackrox/stackrox/blob/master/migrator/migrations/m_212_to_m_213_add_container_start_column_to_indicators/migration_test.go#L37-L40